doc/design/OrionMongoSink.md

```diff
@@ -0,0 +1,110 @@
+#OrionMongoSink
```

Comment:
Maybe some kind of description or reference to the concept of "destination" would be appreciated ;) Mainly if we consider that the the collection name for the collection-per-entity and the collection-per-attribute cases are the same: ```fiware-servicePath_destination``` :)